README.md

```diff
@@ -84,3 +84,8 @@ YourKit supports open source projects with innovative and intelligent tools for
 YourKit is the creator of [YourKit Java Profiler](https://www.yourkit.com/java/profiler/),
 [YourKit .NET Profiler](https://www.yourkit.com/.net/profiler/),
 and [YourKit YouMonitor](https://www.yourkit.com/youmonitor/).
+
+# Folia Inquisitors 
+[<img src="https://github.com/Folia-Inquisitors.png" width=80 alt="Folia-Inquisitors">](https://github.com/orgs/Folia-Inquisitors/repositories)
+[<img src="https://github.com/TechnicallyCoded.png" width=80 alt="TechnicallyCoded">](https://github.com/TechnicallyCoded)
+[<img src="https://github.com/Yomamaeatstoes.png" width=80 alt="Yomamaeatstoes">](https://github.com/Yomamaeatstoes)
```

Comment:
Why was this added? I don't see why we should be specifically calling these groups/people out. Unless you plan to assist in maintaining Folia support, in which case it would be good to mention that here, e.g. "Thanks to <xyz> for maintaining Folia support in WorldEdit". But if you do so, we will assign Folia issues to you.

Reply:
Thanks for the feedback. I have talked to one of the members in the organization who so far plans on doing a pull request. Once that is done it should be reflected in the readme.